From 18c942209969e132f5792aecc7e77e1fadfd30eb Mon Sep 17 00:00:00 2001
From: Nabendu Maiti <1206581+nmaiti@users.noreply.github.com>
Date: Mon, 14 Jul 2025 02:06:24 +0000
Subject: [PATCH 1/2] Add fetures newer first

Signed-off-by: Nabendu Maiti <1206581+nmaiti@users.noreply.github.com>
---
 src/components/sections/featured.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/components/sections/featured.js b/src/components/sections/featured.js
index 1ce4158..2dfdfe4 100644
--- a/src/components/sections/featured.js
+++ b/src/components/sections/featured.js
@@ -352,7 +352,7 @@ const Featured = () => {
 
       <StyledProjectsGrid>
         {featuredProjects &&
-          featuredProjects.map(({ node }, i) => {
+          [...featuredProjects].reverse().map(({ node }, i) => {
             const { frontmatter, html } = node;
             const { external, title, tech, github, cover, cta } = frontmatter;
             const image = getImage(cover);
-- 
2.39.5

